examples/bri-3/src/bri/workgroup/worksteps/persistence/workstepsStorage.agent.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../../../../../prisma/prisma.service';
+import { Workstep } from '../models/workstep';
+
+// Storage Agents are the only places that talk the Prisma language of models.
+// They are always mapped to and from domain objects so that the business layer of the application
+// does not have to care about the ORM.
+@Injectable()
+export class WorkstepStorageAgent extends PrismaService {
+  async getWorkstepById(id: string): Promise<Workstep> {
+    const workstepModel = await this.workstep.findUnique({ where: { id: id } });
```

Comment:
this.workstep is undefined in all methods

Reply:
@skosito Just cloned down a new copy of my branch to check - all I had to do was npm install and it populated the node_modules/.prisma/client/index.d.ts with all the necessary code for both bpiSubject and workstep according to what's in the prisma schema and migrations. @ognjenkurtic Please advise if there is some other step besides migration and generation that need be done in prisma for others to obtain the proper code on install.

Reply:
did not know that it works like that, thanks